Documentation/gettingstarted/aws.rst

```diff
@@ -0,0 +1,303 @@
+guides. Example manifest will create three microservices applications:
+*deathstar*, *tiefighter*, and *xwing*. In this case, we are only going to use
+our *xwing* microservice to secure communications to existing AWS instances.
+
+.. parsed-literal::
+
+    $ kubectl create -f \ |SCM_WEB|\/examples/minikube/http-sw-app.yaml
+    service "deathstar" created
+    deployment "deathstar" created
+    deployment "tiefighter" created
+    deployment "xwing" created
+
+
+Kubernetes will deploy the pods and service in the background.  Running
+``kubectl get pods,svc`` will inform you about the progress of the operation.
+Each pod will go through several states until it reaches ``Running`` at which
+point the pod is ready.
+
+::
+
+    $ kubectl get pods,svc
+    NAME                             READY     STATUS    RESTARTS   AGE
+    po/deathstar-76995f4687-2mxb2    1/1       Running   0          1m
+    po/deathstar-76995f4687-xbgnl    1/1       Running   0          1m
+    po/tiefighter                    1/1       Running   0          1m
+    po/xwing                         1/1       Running   0          1m
+
+    NAME             TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)   AGE
+    svc/deathstar    ClusterIP   10.109.254.198   <none>        80/TCP    3h
+    svc/kubernetes   ClusterIP   10.96.0.1        <none>        443/TCP   3h
+
+Policy Language:
+-----------------
+
+To enforce security to AWS instances, the **ToGroups** rule will be used. This
+rule allows traffic to any cloud provider, in this example, AWS. The example
+policy will be:
+
+.. code-block:: yaml
+
+    ---
+    kind: CiliumNetworkPolicy
+    apiVersion: cilium.io/v2
+    metadata:
+      name: to-groups-sample
+      namespace: default
+    spec:
+      endpointSelector:
+        matchLabels:
+          org: alliance
+          class: xwing
+      egress:
+      - toPorts:
+        - ports:
+          - port: '80'
+            protocol: TCP
+        toGroups:
+        - aws:
+            securityGroupsIds:
+            - 'sg-0fef46100a88d03c3'
+
+This policy allows traffic from pod *xwing* to any AWS instance that is on
+the Security group with ID ``sg-0fef46100a88d03c3``. Also the rule can be by
+instance labels, security groups names or region.
+
+
+Validate that derived policy is in place
+----------------------------------------
+
+Every time that a new policy with ToGroups rules is added a new derivative
+policy will be created, the derivative policy will contain the list of allowed
+IPs in the ToCIDRSet. This new policy will be updated every five minutes to get
+AWS instances changes.
+
+.. parsed-literal::
+
+    $ kubectl get cnp
+    NAME                                                             AGE
+    to-groups-sample                                                 11s
+    to-groups-sample-togroups-044ba7d1-f491-11e8-ad2e-080027d2d952   10s
+
+Where the derivative policy has the following entries on the ToCIDR:
```

Comment:
"Eventually, the derivative policy will contain IPs in the ToCIDR section:"

I'm unsure as to whether "ToCIDR" should be formatted in a specific way.